Create READ.ME

Chaotic-V01D opened the pull request:
READ ME

Chaotic-V01D committed:
Create READ.ME

READ ME

23alevin reviewed (approved):
Apparently needed???

Chaotic-V01D committed:
Methods to access arm extension and rotation dists

Added getters for the distance extended for the arm based on an incrementor, the rate at which the periodic function updates, the rate at which the motor rotates, and the ratio between the motor rotation and the distance extended
Did the same for the arm rotation
TODO: change the constant values when we have them
